Sources/MapboxNavigation/MapView.swift

```diff
@@ -123,9 +123,12 @@ extension MapView {
         guard let layers = try? __map.getStyleLayers().reversed() else { return nil }
         for layer in layers {
             if !(layer.type == "symbol") && !identifiers.contains(layer.id) {
+                let source = try? __map.getStyleLayerProperty(forLayerId: layer.id, property: "source").value as? String
                 let sourceLayer = try? __map.getStyleLayerProperty(forLayerId: layer.id, property: "source-layer").value as? String
                 
-                if let sourceLayer = sourceLayer,
+                if let source = source,
```

Comment:
This code was also removed in most recent change. Please make sure that rebase/merge was correctly performed.

Reply:
Thanks for the suggestion! I'll check this part and correct it.